src/video.cpp

```diff
@@ -611,16 +614,12 @@ namespace video {
         { "recovery_point_sei"s, 0 },
         { "pic_timing_sei"s, 0 },
       },
-      // SDR-specific options
-      {
-        { "profile"s, (int) qsv::profile_hevc_e::main },
-      },
-      // HDR-specific options
-      {
-        { "profile"s, (int) qsv::profile_hevc_e::main_10 },
-      },
-      // Fallback options
+      {},  // SDR-specific options
+      {},  // HDR-specific options
+      {},  // YUV444 SDR-specific options
+      {},  // YUV444 HDR-specific options
```

Comment:
I think we're relying on QSV to automatically determine the best profile by the input pixel format. Is it going to pick a regular HEVC Main profile to encode 4:4:4 on older CPUs that lack 4:4:4 encoding support?

Reply:
I brought back the explicit profile selection through the dictionary option so I think it should be fixed now.